src/user/user.controller.ts

```diff
@@ -19,8 +19,10 @@ export class UserController {
   }
 
   @Patch(':id')
+  @UsePipes(new ValidationPipe())
+  @HttpCode(204)
```

Comment:
Não conheço esse uso do status code assim. Quando acontece o retorno, ele mostra o código igual? Achei interessante. Antes só tinha feito daquele jeito semelhante ao q disse no review passado:
    return res.status(HttpStatus.NO_CONTENT).json(this.userService.update(id, updateUserDto));

Reply:
Eu encontrei esta forma de fazer, achei bem simples e funcionou perfeitamente, não sei qual é a melhor porem até o momento essa esta funcionando perfeitamente nos testes que eu fiz.